src/app.js

```diff
@@ -112,14 +103,14 @@ const app = function() {
     anchor.innerText = blackCard
     li.appendChild(anchor)
     ul.appendChild(li)
-    console.log(blackCard);
   });
 
-  socket.on('selected white cards', function(selectedCards){
-    console.log(JSON.stringify(selectedCards));
-
+  socket.on('all answers played', function(selectedCards){
+    // console.log(JSON.stringify(selectedCards));
 
     const ul = document.querySelector('#selected-white-cards');
+    // Reset the answer board as we're about to redraw
+    ul.innerHTML = '';
```

Comment:
Added this to resolve a bug caused by cumulative appending of cards (e.g. after 3 answers are played, 6 answer cards would appear to have been played)